python/rpdk/java/templates/init/shared/log4j2.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this can be in the data directory and just copied in. no need to render a template with no variables

Reply:
Good point, will do